Source/Scene/Composite3DTileContent.js

```diff
@@ -215,7 +215,7 @@ define([
                 var content = contentFactory(this._tileset, this._tile, this._url);
                 content.initialize(arrayBuffer, byteOffset);
                 this._contents.push(content);
-                contentPromises.push(content.readyPromise);
+                contentPromises.push(content._readyPromise);
```

Comment:
I think this is actually fine how it was, since we want the actual promises and not the deferreds. Both versions work because deferreds are also thennable.